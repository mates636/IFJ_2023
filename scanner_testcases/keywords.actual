11 "String"
32
chyba 1
