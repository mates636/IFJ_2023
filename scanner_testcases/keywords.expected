11 "String"
32
11 "Int?"